<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumeRenderingSettingsPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerVolumeRenderingSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Rendering Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="RenderingMethodLabel">
     <property name="text">
      <string>Default rendering method:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="RenderingMethodComboBox">
     <property name="toolTip">
      <string>Default rendering method</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="QualityControlLabel">
     <property name="text">
      <string>Default quality:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="GPUMemoryLabel">
     <property name="text">
      <string>GPU memory size:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="qSlicerGPUMemoryComboBox" name="GPUMemoryComboBox">
     <property name="toolTip">
      <string>Amount of memory allocated for volume rendering on the graphic card</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="QualityControlComboBox">
     <property name="toolTip">
      <string>Quality control method to def</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="InteractiveSpeedLabel">
     <property name="text">
      <string>Default interactive speed:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkSliderWidget" name="InteractiveSpeedSlider">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>8.000000000000000</double>
     </property>
     <property name="suffix">
      <string> fps</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="SurfaceSmoothingLabel">
     <property name="text">
      <string>Default surface smoothing:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="SurfaceSmoothingCheckBox">
     <property name="toolTip">
      <string>Reduce wood grain artifact to make surfaces appear smoother.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="FramerateLabel_2">
     <property name="text">
      <string>Auto-release resources:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="AutoReleaseGraphicsResourcesCheckBox">
     <property name="toolTip">
      <string>Immediately unload volumes from graphics memory when not visible. Reduces memory usage but makes toggling volume visibility slower.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerGPUMemoryComboBox</class>
   <extends>QComboBox</extends>
   <header>qSlicerGPUMemoryComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
